models/welcome.js

```diff
@@ -0,0 +1,20 @@
+var xtend = Object.assign
+
+module.exports = welcomeModel
+
+function welcomeModel (state, bus) {
+  state.welcome = xtend({
+    show: false
+  }, state.welcome)
+
+  bus.on('repos loaded', function () {
```

Comment:
ohhh, we should probably decide on how to name events; in choo we usually delimit using `:` and camelcase for words. In the `dat` API we use whitespace to space words, and don't think we have namespaces. I feel we should like agree on something and then stick to it. Perhaps `:` to namespace and whitespace to split words?